common/thorhelper/CMakeLists.txt

```diff
@@ -126,6 +133,10 @@ target_link_libraries ( thorhelper
          dalibase
     )
 
+if (USE_PARQUET)
+    target_link_libraries( thorhelper parquetembed )
+endif()
```

Comment:
@ghalliday - On Linux will this cause issues if the parquetembed.so is dynamically loaded/unloaded by a WU?